google/cloud/bigtable/row_set.py

```diff
@@ -119,7 +119,7 @@ def add_row_range_by_prefix_from_keys(self, row_key_prefix):
             :end-before: [END bigtable_row_range_by_prefix_from_keys]
 
         :type row_key_prefix: str
-        :param row_key_prefix: Add all rows that start with this row key prefix.
+        :param row_key_prefix: To retrieve  all rows that start with this row key prefix.
```

Comment:
`To retrieve  all rows that start with this row key prefix.`

The grammar here can be improved.